packages/grafana-runtime/package.json

```diff
@@ -36,5 +37,6 @@
   },
   "resolutions": {
     "@types/lodash": "4.14.119"
-  }
+  },
+  "gitHead": "eef02dc9b15ddd8923ec3571955a47e0f40d6562"
```

Comment:
this should not be in the versioned file, right?  just the published one

Reply:
Yes, good catch